src/ripple/rpc/impl/TransactionSign.cpp

```diff
@@ -374,8 +374,10 @@ transactionPreProcessImpl (
     if (!verify && !tx_json.isMember (jss::Sequence))
         return RPC::missing_field_error ("tx_json.Sequence");
 
-    std::shared_ptr<SLE const> sle = cachedRead(*ledger,
-        keylet::account(srcAddressID).key, ltACCOUNT_ROOT);
+    auto const& key = keylet::account(srcAddressID).key;
```

Comment:
Same here: we know the type of keylet we want. We don't need the ternary at all.